src/components/molecules/UiAlert/UiAlert.vue

```diff
@@ -59,6 +62,10 @@ export interface AlertProps {
    * Use this props to pass attrs for message UiText
    */
   textMessageAttrs?: TextAttrsProps;
+  /**
+   * Use this props to pass HTML attrs
```

Comment:
this comment does not make sense I can pass HTML attrs directly to a component in Vue, without props. 
```
<UiAlert data-testid="alert" />
```